projects/steering/src/steering_control_stalk.c

```diff
@@ -0,0 +1,71 @@
+#include "steering_control_stalk.h"
+#include <stdio.h>
+#include "adc.h"
+#include "adc_periodic_reader.h"
+#include "can.h"
+#include "can_ack.h"
+#include "can_fifo.h"
+#include "can_fsm.h"
+#include "can_hw.h"
+#include "can_msg.h"
+#include "can_rx.h"
+#include "can_transmit.h"
+#include "event_queue.h"
+#include "exported_enums.h"
+#include "fsm.h"
+#include "gpio.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "status.h"
+#include "steering_can.h"
+#include "steering_digital_input.h"
+#include "steering_events.h"
+
+// Will be edited later
+#define STEERING_CONTROL_STALK_LEFT_SIGNAL_VOLTAGE 1000
+#define STEERING_CONTROL_STALK_RIGHT_SIGNAL_VOLTAGE 2000
+#define STEERING_CC_INCREASE_SPEED_VOLTAGE 3000
+#define STEERING_CC_DECREASE_SPEED_VOLTAGE 4000
+#define STEERING_CC_BRAKE_PRESSED_VOLTAGE 5000
+#define TOL 100
+
+// Function prototype
+void control_stalk_callback(uint16_t data, PeriodicReaderId id, void *context);
+
+// Needs to be edited for the actual stalk
+AdcPeriodicReaderSettings reader_settings = { .address = { .port = GPIO_PORT_A, .pin = 3 },
+                                              .callback = control_stalk_callback };
+
+// Stores event id of the event that was just raised
+SteeringAnalogEvent prev = NUM_STEERING_EVENTS;

```

Comment:
Might wanna prefix these as well, goes for anywhere else that I didn't catch this too I guess